### ISSUE TITLE
Maps not working in flatlands and adminworld

### ISSUE THREAD

**Issue by adamvandyck** [TotalFreedom/TotalFreedomMod#1548]
When in either the flatlands or the adminworld, maps don't seem to be working since they are blank and won't show anything from those worlds.

**Comment by Wild1145** [comment]
This is an issue with either Spigot or Vanilla MC. Please consult the Spigot team here: https://www.spigotmc.org/forums/community-feedback-and-suggestions.34/

**Comment by adamvandyck** [comment]
The problem is that on my test server, before TFM 5 was added, maps worked just fine with the latest spigot and after TFM 5 was added, they only worked in custom worlds and not flatlands and adminworld.

**Comment by AvalancheYT** [comment]
@Wild1145, wrong. I updated TFM v4.3 (for chickenfreedom) to 1.9 and everything was fine. It's TFM 5.0.

**Comment by Wild1145** [comment]
@AvalancheYT You know, just saying "wrong" doesnt really make me want to actually give a valid argument back, it makes me want to shout very loudly at you for being so god damn rude.

TFM last time I checked has nothing to do with the maps, so it is entirely possible that maps simply do not support the way that TFM handles worlds, last time I did check, we handle worlds just the same as most other plugins, so it should make no difference. TFM 4.3 and TFM 5.0 are very different plugins, and Spigot 1.8 and 1.9 are very different.

**Comment by JeromSar** [comment]
TFM does not modify map behavior whatsoever.

**Comment by adamvandyck** [comment]
This problem only seems to occur in the worlds that TFM generates like flatlands and adminworld, and other custom worlds that I add, paper maps seem to work fine. Heres a screenshot of the problem 
![2016-03-16_17 34 25](https://cloud.githubusercontent.com/assets/17771884/13830867/b436d38a-eb9d-11e5-948d-cfd041d67d78.png)
